lnwire/commit_sig.go

```diff
@@ -45,6 +46,11 @@ type CommitSig struct {
 	// being signed for. In this case, the above Sig type MUST be blank.
 	PartialSig OptPartialSigWithNonceTLV
 
+	// CustomRecords maps TLV types to byte slices, storing arbitrary data
+	// intended for inclusion in the ExtraData field of the CommitSig
+	// message.
+	CustomRecords CustomRecords
```

Comment:
could just keep this and remove `ExtraData` from CommitSig, as it's not used anywhere

Reply:
`ExtraData` is what ends up being serialized ultimately.